### PR TITLE
feat: add README.md to evaluation templates

### PR DESCRIPTION
**Pull request by qinezh**
Add timestamp to result name, and add instructions to run evaluation and view evaluation results.

### CODE REVIEW

File: evaluation/README.md [hunk 1]
```diff
@@ -1 +1,21 @@
-# Evaluate your app
+# Evaluate your AI app
```
Comment (kimizhu) [inline evaluation/README.md:1]: Give a overall what is evaluation?

File: evaluation/README.md [hunk 2]
```diff
@@ -1 +1,21 @@
...
+
+## Steps to run evaluation
```
Comment (kimizhu) [inline evaluation/README.md:3]: Better to be consistent to the datasets readme.

File: evaluation/README.md [hunk 3]
```diff
@@ -1 +1,21 @@
...
+
+1. Install required libraries with the following command under project root:
+   ```bash
+   pip install -r ai-resource/evaluation/requirements.txt
+   ```
+1. Import your AI app and wrap it into evaluation target (a function named `target_app` in the script). 
+1. If AI-assisted evaluator is selected, set language model's api key in the `.env` file or environment variable.
```
Comment (kimizhu) [inline evaluation/README.md:10]: Do we need to introduce the sdk and built-in evaluator a little bit?

File: evaluation/README.md (hunk 1 above)
Comment (qinezh) [reply evaluation/README.md]: Updated.

File: evaluation/README.md (hunk 3 above)
Comment (qinezh) [reply evaluation/README.md]: Updated.

File: evaluation/README.md (hunk 2 above)
Comment (qinezh) [reply evaluation/README.md]: Updated.

File: evaluation/README.md [hunk 4]
```diff
@@ -1 +1,87 @@
-# Evaluate your app
+# Evaluate Your AI App
+
+## Overall
+
+Evaluation is a critical process to measure the performance, quality and effectiveness of AI apps. This guide will help you to evaluate your AI apps and view the results in AI Toolkit.
```
Comment (supkasar) [inline evaluation/README.md:5]: Evaluation is a critical process to measure the performance, quality, and effectiveness of AI apps. This guide will help you evaluate your AI apps and view the results in AI Toolkit.

File: evaluation/README.md [hunk 5]
```diff
@@ -1 +1,87 @@
...
+
+1. Ensure Python (with version equal to or greater than 3.8) is installed.
+1. Install required libraries with the following command under project root:
+   ```bash
+   pip install -r ai-resource/evaluation/requirements.txt
+   ```
+
+### Step 2: Import your AI app as the evaluation target
+
+Import your AI app in the script and then wrap it into evaluation target. The evaluation target is a function accepts question and return an object of answer.
```
Comment (supkasar) [inline evaluation/README.md:18]: Import your AI app into the script and wrap it in an evaluation target. The evaluation target is a function that takes a question and responds with an answer.

File: evaluation/README.md [hunk 6]
```diff
@@ -1 +1,87 @@
...
+
+```python
+def target_app(question):
+   # import your AI app and wrap it into the evaluation target, like:
```
Comment (supkasar) [inline evaluation/README.md:22]: # import your AI app into the script and wrap it in an evaluation target, like:

File: evaluation/README.md [hunk 7]
```diff
@@ -1 +1,87 @@
...
+Most of the built-in evaluators are AI-assisted:
+- GroundednessEvaluator
+- RelevanceEvaluator
+- CoherenceEvaluator
+- FluencyEvaluator
+- SimilarityEvaluator
+
+If one of those evaluators is selected, you need to set `model_config` with Azure OpenAI model.
+
+> **Note**: `model_config` with OpenAI configuration is not supported for `promptflow-eval` SDK yet and it's planed to be supported in version `v0.3.3`.
```
Comment (supkasar) [inline evaluation/README.md:38]: model_config` with OpenAI configuration isn't supported in the `promptflow-eval` SDK yet, but it's planned for version `v0.3.3`

File: evaluation/README.md [hunk 8]
```diff
@@ -1 +1,87 @@
...
+>
+
+**Use Azure OpenAI Endpoint**:
+1. Create an Azure OpenAI resource and deploy a model. Detailed instructions can be found [here](https://learn.microsoft.com/azure/ai-services/openai/how-to/create-resource).
```
Comment (supkasar) [inline evaluation/README.md:42]: Create an Azure OpenAI resource and deploy a model. Detailed instructions can be found in the [Azure OpenAI documentation] (https://learn.microsoft.com/azure/ai-services/openai/how-to/create-resource).

File: evaluation/README.md [hunk 9]
```diff
@@ -1 +1,87 @@
...
+
+## Additional
+
+### Built-in evaluators
+
+Here list the built-in evaluators from `promptflow-eval`SDK and the meaning of the metrics:
+
+Name | What is this metric?
+| - | - 
+GroundednessEvaluator| Measures how well the model's generated answers align with information from the source data (user-defined context).
```
Comment (supkasar) [inline evaluation/README.md:76]: Measures how closely the model's answers match the source data (user-defined context).

File: evaluation/README.md [hunk 10]
```diff
@@ -1 +1,87 @@
...
+RelevanceEvaluator | Measures the extent to which the model's generated responses are pertinent and directly related to the given questions.
```
Comment (supkasar) [inline evaluation/README.md:77]: Measures how relevant and directly related the model's responses are to the given questions.

File: evaluation/README.md [hunk 11]
```diff
@@ -1 +1,87 @@
...
+CoherenceEvaluator | Measures how well the language model can produce output that flows smoothly, reads naturally, and resembles human-like language.
```
Comment (supkasar) [inline evaluation/README.md:78]: Measures how smoothly and naturally the language model's output reads, and how closely it resembles human language.

File: evaluation/README.md [hunk 12]
```diff
@@ -1 +1,87 @@
...
+FluencyEvaluator | Measures the grammatical proficiency of a generative AI's predicted answer.
+SimilarityEvaluator | Measures the similarity between a source data (ground truth) sentence and the generated response by an AI model.
+F1ScoreEvaluator | Measures the ratio of the number of shared words between the model generation and the ground truth answers.
```
Comment (supkasar) [inline evaluation/README.md:81]: Measures the ratio of shared words between the model's output and the correct answers.

File: evaluation/README.md (hunk 11 above)
Comment (qinezh) [reply evaluation/README.md:78]: same as above.

File: evaluation/README.md (hunk 12 above)
Comment (qinezh) [reply evaluation/README.md:81]: same as above.

**Review by supkasar — APPROVED** [review]

**Pull request merged**